bentoml/_internal/io_descriptors/image.py

```diff
@@ -16,11 +17,15 @@
 from ...exceptions import InternalServerError
 
 if TYPE_CHECKING:
-    import PIL
-    import numpy as np
     import PIL.Image
+    import numpy.typing
+
+    NDArray = numpy.typing.NDArray[t.Any]
```

Comment:
I don't know if this would work for numpy < 1.20

Reply:
it doesn’t have to? We can fix a numpy version for dev env for type checking

Reply:
noted